fe/fe-core/src/main/java/org/apache/doris/plugin/AuditEvent.java

```diff
@@ -60,6 +60,9 @@ public enum EventType {
     public String catalog = "";
     @AuditField(value = "Db")
     public String db = "";
+    // scan table is formatted as {db}.{table} and  {db}.{table}.{index} if table is OlapTable
```

Comment:
maybe you also need modify auditlog plugin， because it will load the log into a doris table. I think you add a new field to auditlog
 and the table schema need change.

Reply:
Thanks for your reply, I will implement this latter